spec/core_functions/color/rgb/four_args/special_functions.hrx

```diff
@@ -282,44 +282,3 @@ a {b: rgb(1, 2, 3, unquote("clamp(0.4, 0.5, 0.6)"))}
 a {
   b: rgb(1, 2, 3, clamp(0.4, 0.5, 0.6));
 }
-
-<===>
-================================================================================
-<===> calc/string/arg_2/args/input.scss
```

Comment:
Those specs were not running at all due to being nested in `calc/string/arg_2/` and `var/arg_2/`.

As they are actually covering the case of special functions in the 2-args signature of the Sass rgb function and not in the 4-args one, I moved them to the proper hrx file to solve the nesting.